db/relationships.yml

```diff
@@ -0,0 +1,68 @@
+relationships:
```

Comment:
Consolidating the values stored in the db, the relationship, and the value we send to the irs into one file is helpful because we can provide a link to program to make sure that what we're defining aligns with their expectations. It also gives us a way to deal with irs_enum values that are NOT in a consistent pattern (which is the case with these newer relationships we're adding)